### ISSUE TITLE
[BUG] Errors launching a local version

### ISSUE THREAD

**Issue by minystreem56** [teoxoy/factorio-blueprint-editor#257]
**Desktop (please complete the following information):**
I started this up on github codespaces but got this error:
<img width="526" alt="image" src="https://github.com/teoxoy/factorio-blueprint-editor/assets/91996030/a618f0aa-8356-4115-9967-b5d4244bb3ae">
Any ideas on how to fix it?
-   OS: Linux (codespaces)
-   Browser chrome, firefox

**Comment by minystreem56** [comment]
No, is this nessasary? I did not want to enter in my factorio info if I didn't have to.

**Comment by minystreem56** [comment]
I will try without entering my info, if that works than I'm just stupid, lol.

**Comment by teoxoy** [comment]
Yeah, it's necessary for now.

**Comment by sighnwaive** [comment]
@teoxoy  should we look into making the exporter optional to run as long as the required data is in place?  I can look at doing that if you want to add a feature to the road map.

**Comment by SM-26** [comment]
@minystreem56 did you get it running? 
I'm also having trouble with the exporter
